mrp_unbuild_subcontracting/views/mrp_unbuild_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.actions.act_window" id="mrp.mrp_unbuild">
+        <field name="domain">[('is_subcontracted', '=', False)]</field>
+    </record>
+
+    <record model="ir.actions.act_window" id="mrp_unbuild_subcontracted">
+        <field name="name">Unbuild Orders - Subcontracted</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">mrp.unbuild</field>
+        <field name="view_mode">tree,kanban,form</field>
+        <field name="domain">[('is_subcontracted', '=', True)]</field>
+        <field name="help" type="html">
+          <p class="o_view_nocontent_smiling_face">
+            No unbuild order found
+          </p><p>
+            An unbuild order is used to break down a finished product into its components.
+          </p>
+        </field>
+    </record>
+
+    <menuitem
+        id="menu_mrp_unbuild_subcontracted"
+        name="Subcontracted Unbuild Orders"
+        parent="mrp.menu_mrp_manufacturing"
+        action="mrp_unbuild_subcontracted"
+        sequence="21"
+        groups="base.group_no_one"
```

Comment:
Why is this menu only available in 'Developer' mode?

Reply:
The same functionality but in the mrp you cannot directly see the generated mrp. I have decided that the unbuilds are hidden like mrps but can be located in developer mode